src/main/java/ui/UI.java

```diff
@@ -174,6 +183,9 @@ private void showMainWindow(String repoId) {
         // Should only be called after panels have been initialized
         ensureSelectedPanelHasFocus();
         initialisePickers();
+        if (shouldCreateSampleBoard && (!TestController.isTestMode() || TestController.isTestStartupBoard())){
```

Comment:
Can the other parts of the conditional be part of `shouldCreateSampleBoard`?

It might be helpful to explain why we don't want to open the sample board when in test mode, but without the flag.

Also, we should move that variable down here, since it's only used here.